src/Auth.ts

```diff
@@ -58,21 +91,45 @@ export class Service {
   // ID of the tenant where the Azure AD app is registered; common if multitenant
   tenant: string;
   cloudType: CloudType = CloudType.Public;
+  // A list of available identities, including the active one
+  availableIdentities?: Identity[];
```

Comment:
Since Service represents a connection and is tied to a single identity, let's move this to Auth